### PR TITLE
Use an absolute path for the xml dir

### PR DESCRIPTION
**Pull request by wesleybalvanz-wf**
@tylerrinnan-wf @seangerhardt-wf @jacobmoss-wf @aldenpeterson-wf

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
Number of Findings: 0

**Comment by jacobmoss-wf** [comment]
+1

**Comment by seangerhardt-wf** [comment]
+1

File: goverge/main.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def _parse_args(argv):
     p.add_argument(
         '--xml_dir',
         action='store',
-        default="xml_reports/",
+        default=os.getcwd() + "/xml_reports/",
```
Comment (aldenpeterson-wf) [inline goverge/main.py]: Probably slightly better to use `os.path.join` here though pretty minor unless you want this to be platform independent

**Pull request merged**